### DISCUSSION TITLE
Stránka s pro děti a rodiče zajímavými dokumenty

### DISCUSSION THREAD

**Discussion by Grejdy** [bosancz/bosan.cz#579] [Ideas]
**Jaký problém tvůj nápad řeší?**
Některé soubory (většinou nějaké dokumenty pdf), co my si běžně můžem najít na g-drivu, běžně mohou používat i děti, ale nemají k nim přístup v elektronické podobě (většinou jim je vedoucí tisknou). Vesměs mě v tomto ohledu napadají hlavně věci související s hvědama (podpisové archy, teoreticky i metodiky, přílohy…), ale mohly by sem spadat i třeba táborové zpěvníky a soubory, co byly na starém webu (seznam s1s2s3 atp.).

**Popiš možnosti řešení**
Nejjednodušší řešení by bylo někam na web hodit odkaz na složku g-drivu s volným přístupem.
Teoreticky by se to dalo vyřešit i elegantněji, a to stejným způsobem, jako přihlášky u akcí (prostě si tam vytvořit jednoduchou stránku s odkazama na stažení a možná třeba datem nahrání a dát do interní sekce nějaké nahrávací rozhraní). To by vypadalo mnohem líp, ale bylo by to o mnoho víc práce (do čehož se asi nikomu chtít nebude:-))

**Comment by jhavelik** [comment]
Nevím jestli je úplně nutné tohle mít na webu. Opravdu si děti někde chtějí hledat papíry k první hvězdě nebo seznam S3?
Pokud to nějaký rodič nebo dítě doopravdy chce co nám brání to poslat elektronicky mailem.
Chápu že to uspokojí nižší jednotky procent dětí otázka jestli nám to stojí za effort (já myslím že ne)

**Reply by Grejdy** [reply]
No, správná otázka, myslím, není jestli chtějí, ale jak by to využívali, kdyby to existovalo.
Jde o to, že máme celkem dost takovýchhle papírů, ke kterým by děti teoreticky mohly mít volný přístup, ale v praxi to funguje spíš tak, že jim to musí člověk dát do packy, aby se k tomu dostaly (nebo aby vůbec zjistily, že něco takového existuje). Konkrétně třeba s1s2s3 jsem už já dostával stylem „tady máš odkaz na starý web (na kterém tím pádem něco takového bylo), mohl by se ti hodit“. A to mi přijde takové polovičaté řešení.
Je to rovna taková věc, o kterou si děti neřeknou prostě proto, že je nenapadne, co by to pro ně mělo za benefit.

**Reply by jhavelik** [reply]
Já v tom ale úplně výhodu. Nedělám si iluze o tom že si děti prohlížejí naše stránky a hledají na nich pdfka která si pak po volných večerech čtou.
Prostě mi přijde jednodušší když si oddilak připraví "onboarding balíček", který pošle řidičovi do mailu. Protože to že někam dáme odkaz na něco co si někdo může stáhnout neznamená že si to odtamtud stáhne.

**Reply by Grejdy** [reply]
Asi by na to dítě nekoukalo jen tak z nudy, ale dovedu si představit, že tam půjde cíleně pro nějakou konkrétní informaci (když není nablízku rodič, nebo rodič k danému dokumentu nemá přístup… ztratil ho, oddílák ho do „balíčku“ nezařadil, nebo mu „balíček“ neposílal vůbec). Navíc mi přijde, že líp působí mít takové věci dostupné na webu, než každému rodičovi zvlášť posílat jeden více či méně sourodý a univerzální balík, který pravděpodobně za čas zapadne kamsi do archivu.

**Comment by jhavelik** [comment]
Btw pojďme "problémy" tohohle typu řešit v Discussions (jiná záložka tady v projektu) a ve chvíli kdy vyhodnotíme že je to feature co chceme dělat tak ji do issues přesunout.

**Reply by Grejdy** [reply]
Vidíš, toho jsem si nevšiml, že tady něco takovýho taky je:-)
